gobblin-modules/gobblin-orc/src/main/java/org/apache/gobblin/writer/GobblinBaseOrcWriter.java

```diff
@@ -183,9 +241,52 @@ public void commit()
       throws IOException {
     closeInternal();
     super.commit();
+    properties.setProp(ORC_WRITER_ESTIMATED_RECORD_SIZE, String.valueOf(estimatedRecordSize));
```

Comment:
Do you plan to persist this value to state store? Seems like we will retry the running every time when container/job restart. How expansive will it be? If not, maybe we can leave with current implementation

Reply:
Ideally we do persist to state store to avoid the initial tuning phase, but currently so far the initial tuning phase is not too expensive so haven't implemented that persistence across runs yet